src/auth/auth.controller.ts

```diff
@@ -2,19 +2,26 @@ import { Body, Controller, Post } from '@nestjs/common';
 import { AuthService } from './auth.service';
 import { JwtToken } from 'common/jwt/jwt.strategy';
 import { AuthCredentialsDto } from './dtos/authCredentialsDto';
-import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { ApiConflictResponse, ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { swaggerApiErrorResponse } from 'common/swagger/swagger-api-error-response';
 
 @ApiTags('Auth')
 @Controller('auth')
 export class AuthController {
   constructor(private authService: AuthService) {}
 
+  @ApiOkResponse({ description: 'User created' })
+  @ApiConflictResponse({
+    description: 'User name already exist',
+  })
+  @swaggerApiErrorResponse()
   @Post('/sign-up')
```

Comment:
it's better to follow the naming convention of decorators, i.e. change s to uppercase S in  @SwaggerApiErrorResponse()